dataactbroker/handlers/fileHandler.py

```diff
@@ -464,6 +471,7 @@ def startGenerationJob(self, submission_id, file_type):
             Tuple of boolean indicating successful start, and error response if False
 
         """
+        valJob = None
```

Comment:
Right, that was from before I pulled the D1/D2 setup into a different function, have now removed this.